app/src/main/java/org/simple/clinic/home/overdue/OverdueAppointment.kt

```diff
@@ -135,6 +136,21 @@ data class OverdueAppointment(
         scheduledAfter: LocalDate
     ): Observable<List<OverdueAppointment>>
 
+    @Query("""
+      SELECT * FROM OverdueAppointment
+      WHERE 
+        appt_facilityUuid = :facilityUuid 
```

Comment:
Weren't we supposed to load overdue list based on patient assigned facility id?

Reply:
For `overdueAtFacility` we have used this `IFNULL(patientAssignedFacilityUuid, appt_facilityUuid) = :facilityUuid`

Reply:
Oh yes, good catch! This screen was done before the assigned facility work was done, so I think I missed updating it while I rerolled. Will fix.